lib/repl.js

```diff
@@ -402,6 +405,16 @@ function REPLServer(prompt,
         });
       } catch (e) {
         debug('parse error %j', code, e);
+        const importErrorStr = 'Cannot use import statement outside a module';
+        if (StringPrototypeIncludes(e.message, importErrorStr)
+        ) {
+          e.message = 'Cannot use import statement inside the Node.js repl, ' +
+            'please use dynamic import instead';
+          const splitStack = StringPrototypeSplit(e.stack, '\n');
+          // console.log(splitStack)
+          splitStack[4] = `SyntaxError: ${e.message}`;
```

Comment:
The stack might even be empty or not a string. We'd have to check more closely for the actual content.

Reply:
when would this be the case? AFAIK this specific error will only happen when static import is used and the consistent and realiable. If V8 ever changed the message our test would fail and we could update implementation at that time.

Thoughts

Reply:
The user could use `Error.stackTraceLimit = 0` or `Error.prepareStackTrace()` for example.

Reply:
Does adding an example of dynamic import a good idea or too verbose for this message?

Reply:
I think it would be too verbose tbqh